schema/url/pass/plugin-pieplot.raml

```diff
@@ -0,0 +1,49 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  PieplotPostRequest:
+    type: VisualizationRequestBase
+    properties:
+      config: PieplotSpec
+  
+  PieplotSpec:
+    additionalProperties: false
+    properties:
+      outputEntityId:
+        type: string
+      showMissingness:
+        type: string
+        enum: ['TRUE', 'FALSE']
```

Comment:
is there a plan for how to represent missingness? Not in the tiny pie view but maybe in the expanded pie view? Like hashing or something? We don't have markers to help distinguish the gray from other colors here.

Reply:
None I know of.. makes the x-axis v overlay question interesting though particularly wrt consistency w other stats charts.. we usually don't provide no data counts for the axes...